statemachine/aws-stepfunctions-statemachine.json

```diff
@@ -170,13 +172,21 @@
   "required": [
     "RoleArn"
   ],
+  "tagging": {
+    "taggable": true,
+    "tagOnCreate": true,
+    "tagUpdatable": true,
+    "cloudFormationSystemTags": true,
+    "tagProperty": "/properties/Tags"
+  },
   "additionalProperties": false,
   "readOnlyProperties": [
     "/properties/Arn",
     "/properties/Name"
   ],
   "createOnlyProperties": [
-    "/properties/StateMachineName"
+    "/properties/StateMachineName",
+    "/properties/StateMachineType"
```

Comment:
Please separate unrelated changes. Changing the update behaviour for this property deserves a separate PR/commit. This wasn't a validation issue as the PR title says, is it?

Reply:
As noted above, separate PR #31 opened for `StateMachineType` changes, and #30 for documentation changes.